assets/cubyz/biomes/rare/crimson_wasteland/black_pit.zig.zon

```diff
@@ -0,0 +1,17 @@
+.{
+	.properties = .{},
+	.minRadius = 16,
+	.maxRadius = 24,
+	.chance = 0,
+	.minHeight = 5,
+	.maxHeight = 10,
+	.roughness = 3,
+	.hills = 3,
+	.stoneBlock = "cubyz:chalk/black",
+	.parentBiomes = .{
+		.{
+			.id = "cubyz:rare/crimson_wasteland/base",
+			.chance = 48,
```

Comment:
I think you forgot to turn the chance here after increasing the radius, the biome is looking rather patchy with all these pits:
<img width="2560" height="1413" alt="Screenshot at 2025-09-24 21-00-06" src="https://github.com/user-attachments/assets/042f749f-dbdb-4e14-934a-277ca9eaaf2b" />